AiTableTopGameMaster.ConsoleApp/Clients/IAgentChatClient.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.SemanticKernel.Agents;
+using Microsoft.SemanticKernel.ChatCompletion;
+
+namespace AiTableTopGameMaster.ConsoleApp.Clients;
+
+/// <summary>
+/// Interface for agent-based chat client that manages conversations using Semantic Kernel's agents framework.
+/// This replaces direct IChatCompletionService usage and enables future multi-agent coordination.
+/// </summary>
+public interface IAgentChatClient
```

Comment:
Shouldn't need to exist. We should be able to do everything in the existing class and interface, but probably with different signatures.